macros/src/main/scala/io/udash/macros/MiscMacros.scala

```diff
@@ -0,0 +1,27 @@
+package io.udash
+package macros
+
+import com.avsystem.commons.macros.AbstractMacroCommons
+
+import scala.reflect.macros.{TypecheckException, blackbox}
+
+class MiscMacros(val ctx: blackbox.Context) extends AbstractMacroCommons(ctx) {
```

Comment:
Would this work if put into `macros/test/main/...`?

Reply:
The most annoying thing is that _neither_ ScalaTest nor Shapeless (`illTyped`) do what I need.